packages/components/buttons/primary-button/src/primary-button.styles.ts

```diff
@@ -23,7 +24,12 @@ const getSizeStyles = (size) => {
   }
 };
 
-const getButtonStyles = (isDisabled, isActive, tone, size) => {
+const getButtonStyles = (
```

Comment:
Wouldn't better to use `Pick`  on `TPrimaryButtonProps` along with union `isActive: Boolean`?

Reply:
That makes sense, and I thought the same.
I didn't want to change the underlying API that much yet however.

Simply migrate. I'll reconsider if this is worth the effort.